packages/dialtone-vue3/prototypes/dialpad/dialpad.stories.js

```diff
@@ -0,0 +1,25 @@
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+import DialpadDefaultTemplate from './dialpad_default.story.vue';
+
+// Story Collection
+export default {
+  title: 'Prototypes/Dialpad',
+  args: {},
+  argTypes: {},
+  excludeStories: /.*Data$/,
+};
+
+// Templates
+const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DialpadDefaultTemplate,
+);
+
+export const Default = {
```

Comment:
Maybe we can rename this `Default` to `Leftbar` then this will be the structure:

<img width="241" alt="image" src="https://github.com/dialpad/dialtone/assets/89984179/fa171a0c-e766-49df-abb0-f2aa24f77f6d">

Just as a suggestion, related to Nina's comment

Reply:
The prototype will be the entire dialpad app, not only the leftbar, right now it only includes the leftbar but will include everything else in the near future.